pom.xml

```diff
@@ -377,6 +426,14 @@
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>findbugs-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
```

Comment:
From my experience, only Java Docs generation takes more time. It doesn't take much time to create the source jar. Having Java Docs plugin is important as we can focus on Java Doc issues before the release time. Having Java Docs plugin enabled by default, developers can focus on Java Docs while development and without having to focus on Java Docs only at the release time.